src/content/resources/tools/FASCSA-order-check.md

```diff
@@ -0,0 +1,19 @@
+---
+title: FASCSA Order Check
+description: Before making any purchase, buyers should review Federal Acquisition Supply Chain Security Act (FASCSA) orders.
+intro: Before making any purchase, buyers should review Federal Acquisition Supply Chain Security Act (FASCSA) orders.
+order: 2
+---
+The federal government issued an [interim rule](https://www.federalregister.gov/documents/2023/10/05/2023-21320/federal-acquisition-regulation-implementation-of-federal-acquisition-supply-chain-security-act) that amends the Federal Acquisition Regulation (FAR) to implement supply chain risk information sharing and FASCSA orders. This rule became effective on December 4, 2023.
+
+## Accessing FASCSA Orders
+[The System for Award Management (SAM)](http://sam.gov) is an official website of the U.S. Government that helps users navigate the federal award lifecycle. 
```

Comment:
This is minor and doesn't have much effect on end users, but I wonder if we should make the link to `https://....` to avoid making the browser redirect from `http` to `https` (same comment of the other link below).

Reply:
Updated links to include https instead of http.